<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UI</class>
 <widget class="QMainWindow" name="UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtRadio</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Spectrum" name="spectrumFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>961</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="Waterfall" name="waterfallFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>961</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>460</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Audio:</string>
    </property>
    <widget class="QComboBox" name="audioComboBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AF Gain</string>
     </property>
    </widget>
    <widget class="QSlider" name="afGainHorizontalSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>231</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Sub RX</string>
    </property>
    <widget class="QPushButton" name="subRxPushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Switch On</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>AF Gain</string>
     </property>
    </widget>
    <widget class="QSlider" name="subRxAfGainHorizontalSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>160</width>
       <height>18</height>
      </rect>
     </property>
     <property name="value">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>460</y>
      <width>401</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Server</string>
    </property>
    <widget class="QComboBox" name="serverHostComboBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>50</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Receiver</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="serverRxSpinBox">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>50</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="serverConnectPushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>963</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBand">
    <property name="title">
     <string>Band</string>
    </property>
    <addaction name="action160"/>
    <addaction name="action80"/>
    <addaction name="action60"/>
    <addaction name="action40"/>
    <addaction name="action30"/>
    <addaction name="action20"/>
    <addaction name="action17"/>
    <addaction name="action15"/>
    <addaction name="action12"/>
    <addaction name="action10"/>
    <addaction name="action6"/>
    <addaction name="actionGen"/>
    <addaction name="actionWWV"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionLSB"/>
    <addaction name="actionUSB"/>
    <addaction name="actionDSB"/>
    <addaction name="actionCWU"/>
    <addaction name="actionCWL"/>
    <addaction name="actionAM"/>
    <addaction name="actionSAM"/>
    <addaction name="actionFMN"/>
    <addaction name="actionDIGL"/>
    <addaction name="actionDIGU"/>
   </widget>
   <widget class="QMenu" name="menuFilter">
    <property name="title">
     <string>Filter</string>
    </property>
    <addaction name="actionFilter_0"/>
    <addaction name="actionFilter_1"/>
    <addaction name="actionFilter_2"/>
    <addaction name="actionFilter_3"/>
    <addaction name="actionFilter_4"/>
    <addaction name="actionFilter_5"/>
    <addaction name="actionFilter_6"/>
    <addaction name="actionFilter_7"/>
    <addaction name="actionFilter_8"/>
    <addaction name="actionFilter_9"/>
   </widget>
   <widget class="QMenu" name="menuNoise_Reduction">
    <property name="title">
     <string>Noise Reduction</string>
    </property>
    <addaction name="actionANF"/>
    <addaction name="actionNR"/>
    <addaction name="actionNB"/>
   </widget>
   <widget class="QMenu" name="menuAGC">
    <property name="title">
     <string>AGC</string>
    </property>
    <addaction name="actionSlow"/>
    <addaction name="actionMedium"/>
    <addaction name="actionFast"/>
    <addaction name="actionLong"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuBand"/>
   <addaction name="menuMode"/>
   <addaction name="menuFilter"/>
   <addaction name="menuNoise_Reduction"/>
   <addaction name="menuAGC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action160">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>160</string>
   </property>
  </action>
  <action name="action80">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>80</string>
   </property>
  </action>
  <action name="action60">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>60</string>
   </property>
  </action>
  <action name="action40">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </action>
  <action name="action30">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </action>
  <action name="action20">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="action17">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>17</string>
   </property>
  </action>
  <action name="action15">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="action12">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action10">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="actionGen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gen</string>
   </property>
  </action>
  <action name="actionWWV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>WWV</string>
   </property>
  </action>
  <action name="actionLSB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>LSB</string>
   </property>
  </action>
  <action name="actionUSB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>USB</string>
   </property>
  </action>
  <action name="actionDSB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DSB</string>
   </property>
  </action>
  <action name="actionCWL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CWL</string>
   </property>
  </action>
  <action name="actionCWU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CWU</string>
   </property>
  </action>
  <action name="actionAM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>AM</string>
   </property>
  </action>
  <action name="actionFMN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FMN</string>
   </property>
  </action>
  <action name="actionFilter_0">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 0</string>
   </property>
  </action>
  <action name="actionFilter_1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 1</string>
   </property>
  </action>
  <action name="actionFilter_2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 2</string>
   </property>
  </action>
  <action name="actionFilter_3">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 3</string>
   </property>
  </action>
  <action name="actionFilter_4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 4</string>
   </property>
  </action>
  <action name="actionFilter_5">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 5</string>
   </property>
  </action>
  <action name="actionFilter_6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 6</string>
   </property>
  </action>
  <action name="actionFilter_7">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 7</string>
   </property>
  </action>
  <action name="actionFilter_8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 8</string>
   </property>
  </action>
  <action name="actionFilter_9">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filter 9</string>
   </property>
  </action>
  <action name="actionANF">
   <property name="text">
    <string>ANF</string>
   </property>
  </action>
  <action name="actionNR">
   <property name="text">
    <string>NR</string>
   </property>
  </action>
  <action name="actionNB">
   <property name="text">
    <string>NB</string>
   </property>
  </action>
  <action name="actionSlow">
   <property name="text">
    <string>Slow</string>
   </property>
  </action>
  <action name="actionMedium">
   <property name="text">
    <string>Medium</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="text">
    <string>Fast</string>
   </property>
  </action>
  <action name="actionLong">
   <property name="text">
    <string>Long</string>
   </property>
  </action>
  <action name="actionSAM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SAM</string>
   </property>
  </action>
  <action name="actionDIGL">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DIGL</string>
   </property>
  </action>
  <action name="actionDIGU">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DIGU</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spectrum</class>
   <extends>QFrame</extends>
   <header>Spectrum.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Waterfall</class>
   <extends>QFrame</extends>
   <header>Waterfall.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
